dataset_test.go

```diff
@@ -61,6 +61,7 @@ func TestDataset(t *testing.T) {
 		}(project, files)
 	}
 	wg.Wait()
-	assert.True(t, len(licenses) >= 766)
-	fmt.Printf("%d %d %d%%\n", len(licenses), len(projects), (100 * len(licenses)) / len(projects))
+	assert.True(t, len(licenses) >= 789)
```

Comment:
I'm getting only 765 on my tests ... what do you think it could be?

Reply:
All sorts of mismatches after the refactoring. The accuracy is fragile as always. You can print the new failures and the old failures and compare by hand. Then trace them.